AK/DOSPackedTime.cpp

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright (c) 2020, the SerenityOS developers.
```

Comment:
Was this file created 3 years ago?

Reply:
Since we are looking to move the definitions for the FAT date and time format into `AK` to share them, and this is very clearly based on the existing structs from `Kernel/FileSystem/FATFS/Definitions.h`, we should probably 'inherit' from that copyright (which is 2022 by Undefine). Feel free to add on yours too, of course, but with the correct year.

Also, please make the FAT implementation in the Kernel use these structs and methods here (if possible), and remove the now-duplicated ones in `Kernel/FileSystem/FATFS/Definitions.h` and `Kernel/FileSystem/FATFS/Inode.cpp`.